src/processing.integration-test.ts

```diff
@@ -31,47 +36,55 @@ describe("processSpace", () => {
   });
 
   it("should process proposals correctly", async () => {
-    const notifyFn = spy();
+    const processProposalsFn = spy();
+    const processAnswersFn = spy();
     const emitter = new EventEmitter();
 
-    const toBlock = 19475121n;
+    const toBlock = oneInchProposalBlockNumber + 1n;
 
     const result = await fn({
       space,
       blockNumber: toBlock,
       calculateBlockRangeFn: () => ({
-        fromBlock: 19475120n,
         toBlock,
```

Comment:
Completely optional, might be a nitpick. can you have fromblock arg first then toblock, it will help with flow of reading.

Reply:
I would love to have named arguments in javascript/typescript, even pay for it.

Reply:
@divyangchauhan reasonable improvement to legibility flow. I'll change that

Reply:
@divyangchauhan done